pages/linux/apk.md

```diff
@@ -0,0 +1,27 @@
+# apk
+
+> Alpine Linux package management tool.
+
+- Update repository indexes from all remote repositories:
+
+`apk update`
+
+- Install a new package:
+
+`apk add {{package}}`
+
+- Remove a package:
+
+`apk del {{package}}`
+
+- Repair package or upgrade it without modifying main dependencies:
+
+`apk fix {{package}}`
+
+- Search package via keyword:
+
+`apk search {{keyword}}`
+
+- List the packages installed in the System:
```

Comment:
"System" should be lowercase. Also, the description says "List the packages installed", but the example has "{{package}}". So does it mean it will show all packages with the given string ? Or is there a mistake ?

Reply:
Looks like `apk info` will list all packages (like the description says) and `apk info {{package}}` only displays for a specific package so I think this may be a mistake.

Reply:
I have made the changes to **apk info** command. Thank you for pointing out.